mantidproject/mantid
# Fix Indirect Corrections Monte Carlo crash

**Description of work.**
This PR fixes a crash into indirect corrections interface. This crash occurred when the user attempted to save the results from the corrections in the Monte Carlo tab.

**Report to:** ISIS/Spencer


**To test:**


Data to test the interface can be found in - #26631.
- Go to Indirect -> Corrections -> Calculate Monte Carlo Absorption
- I've been testing with the following parameters
- Sample - iris26176_graphire002_red.nxs
- Events 1000
- Beam height 3.2, width 2.1
- Shape
- Height 1, width 1, thickness 0.1
- Chemical formula H2-O
- Run and then save results, should no longer be a crash

Fixes #29277. 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
